test/WebJobs.Script.Tests/Binding/HttpBindingTests.cs

```diff
@@ -109,147 +73,36 @@ public void ParseResponseObject_StatusWithNullBody_ReturnsExpectedResult()
 
             object content = null;
             IDictionary<string, object> headers = null;
-            HttpStatusCode statusCode = HttpStatusCode.OK;
+            int statusCode = StatusCodes.Status200OK;
             bool isRawResponse = false;
             HttpBinding.ParseResponseObject(responseObject, ref content, out headers, out statusCode, out isRawResponse);
 
             Assert.Equal(null, content);
-            Assert.Equal(HttpStatusCode.Accepted, statusCode);
-        }
-
-        [Fact]
-        public async Task CreateResultContent_ExpandoObject_ReturnsJsonStringContent()
-        {
-            dynamic expandoObject = new ExpandoObject();
-            expandoObject.name = "Mathew";
-            expandoObject.location = "Seattle";
-
-            StringContent stringContent = HttpBinding.CreateResultContent(expandoObject);
-            string json = await stringContent.ReadAsStringAsync();
-            JObject parsed = JObject.Parse(json);
-            Assert.Equal("Mathew", parsed["name"]);
-            Assert.Equal("Seattle", parsed["location"]);
-            Assert.Equal("text/plain", stringContent.Headers.ContentType.MediaType);
-
-            stringContent = HttpBinding.CreateResultContent(expandoObject, "application/json");
-            json = await stringContent.ReadAsStringAsync();
-            parsed = JObject.Parse(json);
-            Assert.Equal("Mathew", parsed["name"]);
-            Assert.Equal("Seattle", parsed["location"]);
-            Assert.Equal("application/json", stringContent.Headers.ContentType.MediaType);
-        }
-
-        [Fact]
-        public async Task CreateResponse_JsonString_ReturnsExpectedResult()
```

Comment:
I'm guessing this one doesn't apply anymore either?

Reply:
Right. Added a comment. The model changed and we should improve the test coverage to include some of the new methods.